internal/mongo/mongo.go

```diff
@@ -0,0 +1,47 @@
+package mongo
+
+import (
+	"context"
+	"os"
+	"time"
+
+	"github.com/joho/godotenv"
+	"github.com/rs/zerolog/log"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+var (
+	client                   *mongo.Client
+	electionPeriodCollection *mongo.Collection
+	votingCollection         *mongo.Collection
+)
+
+func Init() error {
+
+	err := godotenv.Load()
+	if err != nil {
+		log.Warn().Err(err)
+	}
+
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
```

Comment:
you can add this cancelFunc to the mongo queries

Reply:
how?